packages/service-library/src/servicelib/long_running_tasks/_redis_store.py

```diff
@@ -82,11 +100,9 @@ async def update_task_data(
         *,
         updates: dict[str, Any],
     ) -> None:
-        await handle_redis_returns_union_types(
-            self._redis.hset(
-                self._get_redis_task_data_key(task_id),
-                mapping=_to_redis_hash_mapping(updates),
-            )
+        assert self.hset_if_key_exists is not None  # nosec
+        await self.hset_if_key_exists(  # pylint: disable=not-callable
```

Comment:
please use ```await handle_redis_returns_union_types(``` instead of the pylint. if that is your issue

Reply:
it's not for that, we have the same for the other scripts